mllib/src/main/scala/org/apache/spark/ml/feature/Word2Vec.scala

```diff
@@ -274,6 +274,31 @@ class Word2VecModel private[ml] (
     wordVectors.findSynonyms(word, num)
   }
 
+  /**
+   * Find "num" number of words whose vector representation is most similar to the supplied vector.
+   * If the supplied vector is the vector representation of a word in the model's vocabulary,
+   * that word will be in the results.
+   * @return a tuple of the words list and the cosine similarities list between the synonyms given
+   * word vector.
+   */
+  @Since("2.2.0")
+  def findSynonymsTuple(vec: Vector, num: Int): (Array[String], Array[Double]) = {
```

Comment:
Hi Yanbo, I also tried to put the method here `private` but not working out, since this method `findSynonymsTuple()` is exposed to python client `self._call_java("findSynonymsTuple", word, num)`

or I'm missing something here, I should use `private[feature]` or `private[ml]`? I'm not quite sure here, thanks a lot